### PR TITLE
[test/plugin] immigrate spring async scenario

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
FYI @dmsolr

File: test/plugin/scenarios/spring-async-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            componentId: 2
+            componentName: ''
+            isError: false
+            spanType: Exit
+            peer: skywalking.apache.org:80
+            peerId: 0
+            tags:
+              - {key: url, value: 'http://skywalking.apache.org/?k=v'}
+              - {key: http.method, value: GET}
+          - operationName: /
```
Comment (wu-sheng) [inline test/plugin/scenarios/spring-async-scenario/config/expectedData.yaml]: Maybe the old case is not right either. This span is inside the entry span(`/spring-async-scenario/case/spring-async`), so it is showing async. Please recheck.

Comment (arugal) [reply test/plugin/scenarios/spring-async-scenario/config/expectedData.yaml]: done

**Comment by arugal** [comment]
> This is just for Spring async test, right? I didn't find Spring annotation bean and mvc tests.

Yes，I will provide Spring annotation beans and MVC tests in another PR.

**Comment by wu-sheng** [comment]
@dmsolr  seems we need workload2 and group3 for compose, group4 for new jvm cases

**Pull request merged**